chrome/skin/style-linux.css

```diff
@@ -0,0 +1,4 @@
+#universal-search-recommendation.highlight {
+  background-color: Highlight;
+  color: HighlightText;
+};
```

Comment:
@kaganjd We actually don't want semicolons at the end of CSS blocks, we just want the closing bracket. Here's some discussion on MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax#CSS_declarations_blocks